inst/extdata/modules/drifter/styles.css

```diff
@@ -0,0 +1,15 @@
+.data-container {
+  padding: 0px 20px;
+}
+
+.plot {
```

Comment:
I think that latest changes have width set to 60%

Reply:
Removed that plot css because it was not used